charts/gadget/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{ if .Values.createNamespace }}
```

Comment:
I think this manifest is not necessary. IIRC Helm will complain if target namespace does not exist and `--create-namespace` flag is not specified.